src/main/resources/lib/hudson/test/failed-test.jelly

```diff
@@ -35,28 +35,7 @@ THE SOFTWARE.
     </st:attribute>
   </st:documentation>
     <st:once>
-      <script type="text/javascript">
-      <!-- TODO make sure load doesn't happen every time -->
-      function showFailureSummary(id,query) {
-        var element = document.getElementById(id)
-        element.style.display = "";
-        document.getElementById(id + "-showlink").style.display = "none";
-        document.getElementById(id + "-hidelink").style.display = "";
-
-        if (typeof query !== 'undefined') {
-          var rqo = new XMLHttpRequest();
-          rqo.open('GET', query, true);
-          rqo.onreadystatechange = function() { element.innerHTML = rqo.responseText; }
-          rqo.send(null);
-        }
-      }
-
-      function hideFailureSummary(id) {
-        document.getElementById(id).style.display = "none";
-        document.getElementById(id + "-showlink").style.display = "";
-        document.getElementById(id + "-hidelink").style.display = "none";
-      }
-    </script>
+    <st:adjunct includes="hudson.test.js.failureSummary.js" />
```

Comment:
I am not familiar with Jelly and its syntax.

Reply:
I think you need to use the path suffix that starts right after the folder `src/main/resources`. So `"lib.hudson.test.failureSummary"`. 

(Maybe it would be also good to rename the file to `failure-summary.js`, I'm not sure if the capitalization for files works all over the framework)